micro_dl/utils/aux_utils.py

```diff
@@ -39,6 +39,21 @@ def import_class(module_name, cls_name):
         raise
 
 
+def import_func(module_name, func_name):
```

Comment:
I believe we also have a function for importing a class, could they be combined?

Reply:
Good point!